src/Repository/SubmissionRepository.php

```diff
@@ -33,8 +33,10 @@ public function findFrontPageSubmissions(array $forums, string $sortBy, int $pag
 
         $qb = $this->findSortedQb($sortBy)
             ->where('IDENTITY(s.forum) IN (:forums)')
+            ->andWhere('s.modThread = false')
```

Comment:
this is a possibly security issue - we always have to add s.modThread = false everywhere. 

How about extracting a most common $qb creation into private function, maybe with modThread = false default param and adding all-caps warning - IF YOU FUCK THIS UP NORMAL USERS WILL SEE MOD POSTS

Reply:
Alright, I can pull that into a different function